support/Pipelines/Bbh/ControlId.py

```diff
@@ -149,7 +149,7 @@ def control_id(
     radial_expansion_velocity = binary_data["Expansion"]
 
     # File to write control diagnostic data
-    data_file = open(f"{id_run_dir}/ControlParamsData.txt", "w")
+    data_file = open(f"{id_run_dir}/../ControlParamsData.txt", "w")
```

Comment:
@nilsvu Should we avoid relative paths and just pass two arguments? 1) The control params directory, 2) the directory that holds all the control params dirs

Reply:
I think this is fine, it's confined to the control loop with no public interface that can go wrong